scripts/diffThemes.ts

```diff
@@ -0,0 +1,66 @@
+import {PrimerStyleDictionary} from '~/src/PrimerStyleDictionary'
+// @ts-expect-error: no types exists for style-dictionary createDictionary
+import createDictionary from 'style-dictionary/lib/utils/createDictionary'
+import {themes as themesConfigArray} from './themes.config'
+import type StyleDictionary from 'style-dictionary'
+
+PrimerStyleDictionary.registerTransform({
```

Comment:
Should this be here? I thought we put our transforms in a dedicated folder or is this a transient config change only needed for the diff?

Reply:
Actually, good spotting, this is a leftover, I ended up not needing it.

It used to be for this diff only though.